### PR TITLE
Update/add zendesk site

### PR DESCRIPTION
**Pull request by souredoutlook**
- this PR adds types and a transformer for a new Algolia Index
- this PR updates the snapshots to accommodate the new index

### CODE REVIEW

File: src/sentry-global-search/lib/types.ts [hunk 1]
```diff
@@ -44,6 +44,13 @@ export type SearchHit = ObjectWithObjectID & {
    */
   _highlightResult?: { [k: string]: { value: string } };
   _snippetResult?: { [k: string]: { value: string } };
+  /**
+   * String representing secondary context information.
```
Comment (JonasBa) [inline src/sentry-global-search/lib/types.ts:48]: This is an obj now

File: src/sentry-global-search/lib/types.ts [hunk 2]
```diff
@@ -44,6 +44,13 @@ export type SearchHit = ObjectWithObjectID & {
...
+   */
+  category?: {
+    id?: string,
```
Comment (JonasBa) [inline src/sentry-global-search/lib/types.ts:51]: I would just add a comment with an example category here, it makes it easy for whoever is reading this to understand what it represents.

File: src/sentry-global-search/lib/types.ts (hunk 1 above)
Comment (souredoutlook) [reply src/sentry-global-search/lib/types.ts]: fixed in edcd2d58ebde045da2e32e9d90088b2f9e802432

File: src/sentry-global-search/lib/types.ts (hunk 2 above)
Comment (souredoutlook) [reply src/sentry-global-search/lib/types.ts:53]: fixed in edcd2d58ebde045da2e32e9d90088b2f9e802432

**Pull request merged**